checks/linux/password_manager.go

```diff
@@ -77,33 +77,26 @@ func checkForBrowserExtensions() bool {
 }
 
 func isPackageInstalled(pkgName string) bool {
+	output, err := shared.RunCommand("which", pkgName)
+	if err == nil && !strings.Contains(output, "not found") {
+		log.Debug("Package found in PATH: " + pkgName)
+		return true
+	}
+
 	pkgManagers := make(map[string]string)
 
 	// Check which package managers are available
-	if _, err := shared.RunCommand("which", "dpkg"); err == nil {
-		pkgManagers["apt"] = "dpkg -l"
-		log.Debug("apt package manager found")
-	}
+	// We only need to check for package managers that don't inject
+	// the binaries into the $PATH. Those that do are not needed
+	// because we always first check the $PATH with `which`
```

Comment:
Binary names sometimes aren't same as package names.

In case of 1password on nixos:
```
> nix-shell -p _1password-cli _1password-gui --run "which 1password"
<No output>

> nix-shell -p _1password-cli _1password-gui --run "which op"
/nix/store/9qpcy6cjrlagj1gcbpfppbinp1ajh495-1password-cli-2.30.3/bin/op
```

Reply:
Fair point. Shall we do both then? First `which`, then all package managers?

If yes, a question about nix-store lookup: nix store contains a ton of currently non-installed stuff. Does that query only search for stuff in `/run/current-system/`?

Reply:
and for containers name can be com.one.1pass :D